scripts/install/setup_properties.sh

```diff
@@ -95,7 +95,7 @@ export GKE_CLUSTER=${GKE_CLUSTER:-\$DEPLOYMENT_NAME}
 export ZONE=$ZONE
 export REGION=$REGION
 
-export SPINNAKER_VERSION=1.14.3
+export SPINNAKER_VERSION=1.14.5
```

Comment:
Out of curiosity - how do we test changes like this?

Reply:
We manually kick off a very simple test: https://github.com/GoogleCloudPlatform/spinnaker-for-gcp/blob/master/apptest/tester/build-and-run-tests.sh

It checks that clouddriver comes up healthy and that the default account is available.

Reply:
Now that we have a cleanup script, would it make sense to think about configuring a nightly integration test, or something like that?